assets/src/edit-story/components/richText/karma/multiSelection.karma.js

```diff
@@ -0,0 +1,254 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import { waitFor } from '@testing-library/react';
+
+/**
+ * Internal dependencies
+ */
+import { Fixture } from '../../../karma';
+import { initHelpers } from './_utils';
+
+describe('CUJ: Editor Can Style Text', () => {
+  describe('Action: Styling text in multiple text fields', () => {
+    const data = {};
+
+    const {
+      getTextContent,
+      addInitialText,
+      selectBothTextFields,
+      selectTextField,
+    } = initHelpers(data);
+
+    beforeEach(async () => {
+      data.fixture = new Fixture();
+      await data.fixture.render();
+
+      // Add text box + extra
+      await addInitialText(true);
+    });
+
+    afterEach(() => {
+      data.fixture.restore();
+    });
+
+    it('should both have the correct initial text and formatting', () => {
+      // Get content of first textfield
+      expect(getTextContent(0)).toBe('Fill in some text');
+      // Get content of second textfield
+      expect(getTextContent(1)).toBe('Number #2');
+    });
+
+    it('should apply formatting correctly for identically styled text fields', async () => {
+      const {
+        bold,
+        italic,
+        underline,
+        fontWeight,
+        letterSpacing,
+        fontColor,
+      } = data.fixture.editor.inspector.designPanel.textStyle;
+
+      // Select both text fields
+      await selectBothTextFields();
+
+      // Check all styles are default
+      expect(bold.checked).toBe(false);
+      expect(italic.checked).toBe(false);
+      expect(underline.checked).toBe(false);
+      expect(fontWeight.value).toBe('Regular');
+      expect(letterSpacing.value).toBe('0%');
+      expect(fontColor.hex.value).toBe('000000');
+
+      // Toggle all styles
+      await data.fixture.events.click(italic.button);
```

Comment:
Would be nice to use a different font family somewhere as well.

Reply:
Font family is not an inline style. Yes, non-inline styles should be tested too, but not here :)

Reply:
I know it's not an inline style, however, it's related -- e.g. to font weights. If you think it's not related to how the font weights are overwritten in case of different font families, then let's ignore.

Reply:
Oh, we're not doing that yet anyway, so nothing happens to font weights when you change font family currently. That's to be done in #1899.